test/validate.txt

```diff
@@ -0,0 +1,14 @@
+Validating format...
+Format validation succeeded!
+Validating endpoints...
+VALID immutable/git_branch (git) 
+VALID immutable/git_hash (git) 
+Remote repository validated but requested version is not ref so unable to verify.
+VALID immutable/git_tag (git) 
+VALID immutable/hg_branch (hg) 
+VALID immutable/hg_hash (hg) 
+VALID immutable/hg_tag (hg) 
+VALID immutable/svn_ver (svn) 
+VALID immutable/tar_hash (tar) 
+VALID immutable/zip_hash (zip) 
+Endpoint validation succeeded!
```

Comment:
The way `validate` outputs information doesn't match the structure / style used by the other commands.